src/components/10-atoms/text/index.js

```diff
@@ -19,47 +19,14 @@ class AXAText extends NoShadowDOM {
 
   static get properties() {
     return {
-      variant: { type: String },
+      variant: { type: String, reflect: true },
     };
   }
 
   constructor() {
     super();
     applyDefaults(this);
   }
-
-  render() {
```

Comment:
@markus-walther why are you removing this? as a reminder for axa text: AXA Text should always print semantically correct HTML, ergo text should alwys be wrapped via a `span` or a `p` tag. Removing this logic means that you will entirely rely on the user that he surrounds the text via a `span` or a `p` tag. The deleted logic takes care of that. Did this logic break anything or did u solve the requirement that `<axa-text>Text</axa-text>` and  `<axa-text><span>something</span></axa-text>` both create a TEXT based HTML tag once?

Reply:
axa-text is itself the wrapper. The deleted logic was brittle and lead to the bug ticket that this one fixes, elegantly.

Reply:
@markus-walther requirement must be full filled. If a customer does not wrap its text in a p or span tag, we need to make sure is semantically correct. As idea, set role=“paragraph” IF direct child IS not a span or a p tag

Reply:
Done